### PR TITLE
Added handling for multiple types for the same indicator.

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19417

## Description
When an indicator received multiple types in DBot Score and the indicator was given a manual reputation, it was not ignored in the script.
Now, When uniting all arrays to one, and removing from there, it works as expected.

### CODE REVIEW

File: TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
 id: Test-IsMaliciousIndicatorFound
-version: -1
+version: 17
```
Comment (anara123) [inline TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml]:
```suggestion
version: -1
```

**Comment by anara123** [comment]
@teizenman test playbook failing

**Comment by teizenman** [comment]
@kirbles19 Could you please go over the change log at: https://github.com/demisto/content/blob/IsMaliciousIndicatorFound/Scripts/script-IsMaliciousIndicatorFound_CHANGELOG.md

**Comment by kirbles19** [comment]
@teizenman Done.

**Comment by teizenman** [comment]
@kirbles19 Thanks!

**Pull request merged**